### PR TITLE
Repl: Make it easier to heapdump processes

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

**Comment by mia-pi-git** [comment]
Oops.

File: lib/repl.ts [hunk 1]
```diff
@@ -42,6 +42,17 @@ export const Repl = new class {
 		if (!process.listeners('SIGINT').length) {
 			process.once('SIGINT', () => process.exit(128 + 2));
 		}
+		(global as any).heapdump = (targetPath?: string) => {
+			if (!targetPath) targetPath = `${filename}-${Date.now()}`;
```
Comment (Zarel) [inline lib/repl.ts:46]:
```suggestion
			if (!targetPath) targetPath = `${filename}-${new Date().toISOString()}.heapsnapshot`;
```

Comment (mia-pi-git) [reply lib/repl.ts:46]: (heapsnapshot is automatically appended)

Comment (Zarel) [reply lib/repl.ts:46]: (edited)

**Comment by Zarel** [comment]
Looks good, thanks!

**Pull request merged**